composer.json

```diff
@@ -31,7 +31,8 @@
         "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
         "symfony/phpunit-bridge": "^5.3",
         "symfony/routing": "^4.4 || ^5.0 || ^6.0",
-        "twig/twig": "^1.40 || ^2.9 || ^3.0"
+        "twig/twig": "^1.40 || ^2.9 || ^3.0",
+        "phpstan/phpstan": "^1.4"
```

Comment:
So far, the dependencies were sorted alphabetically. You're about to change that. 🙃

Reply:
yeah, they were sorted, but without the config telling `composer require` to apply the sorting. Let me change that.